MAINTAINERS.md

```diff
@@ -84,3 +84,177 @@ Submit a PR with the changes and request a core team review.
 
 Update the list of active branches in the `hacking/tagger/tag.py` script on the `devel` branch.
 Add the new stable branch and remove the lowest version from the `DEFAULT_ACTIVE_BRANCHES` tuple.
+
+## Updating the package doc builds repository
+
+The `ansible-community/package-doc-builds` repository holds the deployed output of package doc builds.
+For each version of the Ansible package that is currently supported, there needs to be a corresponding branch in the `ansible-community/package-doc-builds` repository.
+
+Before the docs for new package version can be built on Read the Docs, someone needs to create a branch as follows:
+
+```bash
+# Make sure your checkout is up to date.
+git fetch upstream
+
+# Create the new package version branch against the devel branch.
+git checkout -b 11 upstream/devel
+
+# Push the new package version branch to the repository.
+git push upstream 11
+```
+
+## Updating the package docs build workflow
+
+The build workflow lists the Ansible package versions as input options, for example:
+
+```yaml
+options:
+- devel
+- '11'
+- '10'
+- '9'
+```
+
+After creating a new branch for the Ansible package version in the `ansible-community/package-doc-builds` repository, you should check that the build workflow lists that version as an option.
+If the version is not listed, open a PR against the `devel` branch to update the workflow file.
+
+## Maintaining Read the Docs projects
+
+There are two Read the Docs projects associated with the `ansible-documentation` repository, as follows:
+
+* Ansible package
+* Ansible core
+
+### Configuration files
+
+The Ansible package and Ansible core projects use separate configuration files, as follows:
+
+* Ansible package configuration is the `package-doc-builds` repository at [https://github.com/ansible-community/package-doc-builds/blob/devel/.readthedocs.yaml](https://github.com/ansible-community/package-doc-builds/blob/devel/.readthedocs.yaml)
+* Ansible core configuration is in the `ansible-documentation` repository at [https://github.com/ansible/ansible-documentation/blob/devel/.readthedocs.yaml](https://github.com/ansible/ansible-documentation/blob/devel/.readthedocs.yaml)
+
+> Each branch has its own Read the Docs configuration file.
+
+Most of the time there is no need to update the Read the Docs configuration files.
+The only change that should be necessary between versions is the Python version, for example:
+
+```yaml
+build:
+  os: ubuntu-22.04
```

Comment:
shouldn't we also need to update the ubuntu version now and then?

Reply:
yeah, I suppose we could also just change this to `ubuntu-latest` and not worry about it. fair point though. I think latest will be fine for these RTD configs because they are quite minimal and use only standard shell commands.

Reply:
my bad I guess. this was already `ubuntu-lts-latest`.